app/assets/javascripts/basetypes/Any.js

```diff
@@ -10,20 +10,19 @@ function RecursiveCast(any) {
   if (any && any.value && any.unit) {
     return new cql.Quantity(any);
   }
-  if (any && any.code && any.codeSystem) {
+  if (any && any.code && (any.codeSystemOid || any.system)) {
```

Comment:
I understand that you are switching out codeSystem for codeSystemOid, but where is `any.system` coming from, and why is that a valid alternative now but before we weren't checking for `any.system` if `any.codeSystem` was null?

Reply:
Previously in basetypes/Code.js we were checking `{ code: code.code, codeSystem: code.codeSystem || code.system };`  I think this may have just been missing here before